ci/rstudio-connect/install/basic-values.yaml

```diff
@@ -0,0 +1,2 @@
+readinessProbe:
+  enabled: false
```

Comment:
seems bad that the basic install requires disabling the readiness probe.
Add an issue to fix this?

Reply:
I'll add an issue for each product, to ensure they can install and not fail with an empty values file

Reply:
https://github.com/rstudio/helm/issues/587